src/main/resources/assets/caverns_and_chasms/lang/ru_ru.json

```
@@ -0,0 +1,810 @@
+
+  "block.caverns_and_chasms.zirconia_block": "Фианитовый блок",
+  "item.caverns_and_chasms.zirconia": "Фианит",
+  "item.caverns_and_chasms.music_disc_copy": "Скопированная пластинка",
+  "item.caverns_and_chasms.music_disc_copy.desc": "Копия:",
+
+  "item.caverns_and_chasms.spinel": "Шпинель",
+  "block.caverns_and_chasms.spinel_block": "Шпинелевый блок",
+  "block.caverns_and_chasms.spinel_bricks": "Шпинелевые кирпичи",
+  "block.caverns_and_chasms.spinel_brick_stairs": "Ступеньки из шпинелевого кирпича",
+  "block.caverns_and_chasms.spinel_brick_slab": "Плита из шпинелевого кирпича",
+  "block.caverns_and_chasms.spinel_brick_wall": "Ограда из шпинелевого кирпича",
+  "block.caverns_and_chasms.spinel_pillar": "Шпинелевая колонна",
+  "block.caverns_and_chasms.spinel_lamp": "Шпинелевый фонарь",
+  "block.caverns_and_chasms.tmt": "Хиломит",
+  "item.caverns_and_chasms.tmt_minecart": "Вагонетка с хиломитом",
+  "item.caverns_and_chasms.bejeweled_pearl": "Украшенный жемчуг",
+  "item.caverns_and_chasms.bejeweled_apple": "Украшенное яблоко",
+  "item.caverns_and_chasms.blunt_arrow": "Тупая стрела",
+
+  "item.caverns_and_chasms.turquoise": "Бирюза",
+  "block.caverns_and_chasms.turquoise_block": "Бирюзовый блок",
+  "block.caverns_and_chasms.turquoise_pillar": "Бирюзовая колонна",
+  "block.caverns_and_chasms.turquoise_tile_slab": "Плита из бирюзового плитняка",
+  "block.caverns_and_chasms.turquoise_tile_stairs": "Ступеньки из бирюзового плитняка",
+  "block.caverns_and_chasms.turquoise_tile_wall": "Ограда из бирюзового плитняка",
+  "block.caverns_and_chasms.turquoise_tiles": "Бирюзовый плитняк",
+  "item.caverns_and_chasms.monocle": "Монокль",
+  "block.caverns_and_chasms.caviar": "Икра",
+
+  "item.caverns_and_chasms.living_flesh": "Живая плоть",
+  "block.caverns_and_chasms.sanguine_block": "Сангвинический блок",
+  "block.caverns_and_chasms.sanguine_tiles": "Сангвинический плитняк",
+  "block.caverns_and_chasms.sanguine_tile_stairs": "Ступеньки из сангвинического плитняка",
+  "block.caverns_and_chasms.sanguine_tile_wall": "Ограда из сангвинического плитняка",
+  "block.caverns_and_chasms.sanguine_tile_slab": "Плита из сангвинического плитняка",
+  "block.caverns_and_chasms.fortified_sanguine_tiles": "Укреплённый сангвинический плитняк",
+  "block.caverns_and_chasms.fortified_sanguine_tile_stairs": "Ступеньки из укреплённого сангвинического плитняка",
+  "block.caverns_and_chasms.fortified_sanguine_tile_wall": "Ограда из укреплённого сангвинического плитняка",
+  "block.caverns_and_chasms.fortified_sanguine_tile_slab": "Плита из укреплённого сангвинического плитняка",
+  "item.caverns_and_chasms.sanguine_helmet": "Сангвинический шлем",
+  "item.caverns_and_chasms.sanguine_chestplate": "Сангвинический нагрудник",
+  "item.caverns_and_chasms.sanguine_leggings": "Сангвинические поножи",
+  "item.caverns_and_chasms.sanguine_boots": "Сангвинические ботинки",
+
+  "item.caverns_and_chasms.necromium_nugget": "Кусочек некромиума",

```

Возможно, стоит изменить на "Кусочек **некромия**", чтобы соответствовало изделиям из него ниже?
```suggestion
  "item.caverns_and_chasms.necromium_nugget": "Кусочек некромия",

```